### PR TITLE
latest changes

### LINKED ISSUE #28: Make proper footer
- [x] brief description of what skeeep is
- [x] add skeeep and WEEE Open logo ("made by WEEE Open")
- [x] link to this repository
- [x] link to general skeeell page on the team's website with explanation of the project and contact to let people ask us to add their software to skeeep

Everything translated to:
- [x] English (e-caste)
- [x] Italian (e-caste)
- [x] Persian (hesam1993)
- [x] Simplified Chinese (itIsJim)

### PR DESCRIPTION
**Pull request by hesam1993**

### CODE REVIEW

File: src/Components/Translations.js [hunk 1]
```diff
@@ -94,10 +94,7 @@ const translations = {
     },
   },
   fa: {
-    footer: {
-      firstLine: "محل قرار گیری فوتر",
-      secondLine: "TODO",
-    },
+    footer: "مجموعه ای از منابع مفید برای دانشجویان، ساخته شده توسط ", // TODO: translate
```
Comment (e-caste) [inline src/Components/Translations.js]: We need one translation for the first line of the footer and one for the second line.

**Review by e-caste — CHANGES_REQUESTED** [review]
Please also remove the changed package-lock.json from your PR

File: src/Components/Translations.js (hunk 1 above)
Comment (hesam1993) [reply src/Components/Translations.js]: please check if they are correct now

File: src/Components/Translations.js [hunk 2]
```diff
@@ -95,9 +95,9 @@ const translations = {
   },
   fa: {
     footer: {
-      firstLine: "محل قرار گیری فوتر",
-      secondLine: "TODO",
-    },
+      firstLine: "مجموعه ای از منابع مفید برای دانشجویان، ساخته شده توسط ",
+      secondLine: "برای اطلاعات بیشتر کلیک کنید",
+    }, // TODO: translate
```
Comment (e-caste) [inline src/Components/Translations.js]:
```suggestion
    },
```

**Review by e-caste — APPROVED** [review]

**Pull request merged**